src/sync/sync_manager.cpp

```diff
@@ -286,25 +287,21 @@ bool SyncManager::perform_metadata_update(std::function<void(const SyncMetadataM
 }
 
 std::shared_ptr<SyncUser> SyncManager::get_user(const std::string& identity,
```

Comment:
Note that this method signature change might be perilous, since it doesn't break existing bindings (but the arguments they pass in will not be in the right order anymore). I may change this.